### DISCUSSION TITLE
Welcome to MyRubyDraft Discussions!

### DISCUSSION THREAD

**Discussion by Richard0307** [Richard0307/MyRubyDraft#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Richard0307** [comment]
At that stage, the mock interview pages have some bugs which will be fixed in next week

**Comment by hu1132889049** [comment]
Any thing else problems with this project?

**Reply by Richard0307** [reply]
No,that's all work now!
